src/styles/index.scss

```diff
@@ -1,3 +1,13 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
+
+@import './blocks/card';
+@import './blocks/stars';
```

Comment:
Consider creating a `variables.scss` file for SCSS variables and import it here to enhance maintainability and consistency across styles.